crates/ruff_python_formatter/tests/snapshots/black_compatibility@simple_cases__fmtpass_imports.py.snap

```diff
@@ -0,0 +1,93 @@
+---
+source: crates/ruff_python_formatter/tests/fixtures.rs
+input_file: crates/ruff_python_formatter/resources/test/fixtures/black/simple_cases/fmtpass_imports.py
+---
+## Input
+
+```py
+# Regression test for https://github.com/psf/black/issues/3438
+
+import ast
+import collections  # fmt: skip
+import dataclasses
+# fmt: off
+import os
+# fmt: on
+import pathlib
+
+import re  # fmt: skip
+import secrets
+
+# fmt: off
+import sys
+# fmt: on
+
+import tempfile
+import zoneinfo
+```
+
+## Black Differences
+
+```diff
+--- Black
++++ Ruff
+@@ -3,6 +3,7 @@
+ import ast
+ import collections  # fmt: skip
+ import dataclasses
++
+ # fmt: off
```

Comment:
This deviates from Black, but I feel like Black is wrong here?

Given:
```python
import dataclasses
# fmt: off
import os
# fmt: on
```

Why omit the empty line after `import dataclasses`? (Black would insert an empty line for a regular comment.)